pkg/controllers/user/pipeline/controller/pipelineexecution/deploy.go

```diff
@@ -32,8 +33,11 @@ import (
 	"k8s.io/kubernetes/pkg/credentialprovider"
 )
 
-const projectIDFieldLabel = "field.cattle.io/projectId"
-const defaultPortRange = "34000-35000"
+const (
+	projectIDFieldLabel        = "field.cattle.io/projectId"
+	defaultPortRange           = "34000-35000"
+	resourceQuotaInitCondition = "ResourceQuotaInit"
```

Comment:
can we make the variable global in pkg/controllers/user/resourcequota/resource_quota_sync.go instead of redefining?

Reply:
Updated